src/components/SnapshotDetails/SnapshotDetailsView.tsx

```diff
@@ -0,0 +1,161 @@
+  );
+
+  const [commitStatus] = useCommitStatus(applicationName, commit.sha);
+
+  if (loadErr || (loaded && !snapshot)) {
+    return (
+      <ErrorEmptyState
+        httpError={HttpError.fromCode(loadErr ? (loadErr as any).code : 404)}
+        title="Snapshot not found"
+        body="No such snapshot"
+      />
+    );
+  }
+
+  if (!plrLoadError && !plrLoaded) {
+    return <LoadingBox />;
+  }
+
+  if (snapshot?.metadata) {
+    return (
+      <DetailsPage
+        headTitle={snapshot.metadata.name}
+        breadcrumbs={[
+          ...applicationBreadcrumbs,
+          {
+            path: `/application-pipeline/workspaces/${workspace}/applications/${applicationName}/snapshots/${snapshotName}`,
+            name: snapshot.metadata.name,
+          },
+        ]}
+        title={
+          <>
+            <Text component={TextVariants.h2} data-test="snapshot-name">
+              {snapshotName}
+            </Text>
+            {commit?.sha && (
+              <>
+                <Text component={TextVariants.p} data-test="snapshot-header-details">
+                  Triggered by {commit.shaTitle}{' '}
+                  <CommitLabel
+                    gitProvider={commit.gitProvider}
+                    sha={commit.sha}
+                    shaURL={commit.shaURL}
+                  />{' '}
+                  at{' '}
+                  <Timestamp
+                    timestamp={snapshot.metadata.creationTimestamp}
+                    className="pf-u-display-inline"
+                  />
+                </Text>
+                <Text component={TextVariants.p} data-test="snapshot-commit-label">
+                  <StatusIconWithTextLabel status={commitStatus as runStatus} />
+                </Text>
+              </>
+            )}
+          </>
+        }
+        actions={[
+          {
+            key: 'edit',
+            label: 'Edit',
+            component: (
+              <Link
+                to={`/application-pipeline/workspaces/${workspace}/applications/${applicationName}/snapshots/${snapshotName}/edit`}
+              >
+                Edit
+              </Link>
+            ),
+            isDisabled: !canUpdateSnapshots,
+            disabledTooltip: "You don't have access to edit this integration test",
+          },
+        ]}
+        baseURL={`/application-pipeline/workspaces/${workspace}/applications/${applicationName}/snapshots/${snapshotName}`}
+        tabs={[
+          {
+            key: 'overview',
+            label: 'Overview',
+            isFilled: true,
+            component: (
+              <SnapshotOverviewTab
+                snapshot={snapshot}
+                commit={commit}
+                buildPipelineName={buildPipelineName}
+              />
+            ),
+          },
+          {
+            key: 'pipelineruns',
+            label: 'Pipeline runs',
+            component: <SnapshotOverviewTab snapshot={snapshot} commit={commit} />,
+          },
+        ]}
```

Comment:
Both tabs are showing the same page, is it expected?
![0](https://github.com/openshift/hac-dev/assets/20013884/85fcfb12-037f-48ea-8e3b-d13ef21fad9b)
![1](https://github.com/openshift/hac-dev/assets/20013884/d1e7720d-04fb-4341-aeac-5d62ca612d74)

Reply:
removed the PipelineRun tab for now

Reply:
<img width="1505" alt="Screenshot 2023-08-17 at 5 09 06 PM" src="https://github.com/openshift/hac-dev/assets/24852534/5243534a-c938-4ab2-b028-f23c842ed17b">
<img width="1442" alt="Screenshot 2023-08-17 at 5 09 14 PM" src="https://github.com/openshift/hac-dev/assets/24852534/577894f2-607c-4a7e-8f8c-bd3386c0a167">